src/snyk/base/modules/snykLib.ts

```diff
@@ -51,8 +51,19 @@ export default class SnykLib extends BaseSnykModule implements ISnykLib {
   }
 
   async setupFeatureFlags(): Promise<void> {
-    const isEnabled = await this.featureFlagService.fetchFeatureFlag(FEATURE_FLAGS.consistentIgnores);
+    let isEnabled = await this.featureFlagService.fetchFeatureFlag(FEATURE_FLAGS.consistentIgnores);
     configuration.setFeatureFlag(FEATURE_FLAGS.consistentIgnores, isEnabled);
+
+    try {
+      isEnabled = await this.featureFlagService.fetchFeatureFlagWithoutErrorHandling(
+        FEATURE_FLAGS.snykCodeInlineIgnore,
+      );
+    } catch (err) {
+      // if we cannot get the feature flag, we default to true for
+      // minimal disruption of the existing customer feature
+      isEnabled = true;
+    }
+    configuration.setFeatureFlag(FEATURE_FLAGS.snykCodeInlineIgnore, isEnabled);
   }
```

Comment:
I just saw that if we provide a fallback to `fetchFeatureFlag` as commented before and we don't mind concurrency, instead of allSettle we go:

```ts
  async setupFeatureFlags(): Promise<void> {
    const flags = [
      { flag: FEATURE_FLAGS.consistentIgnores, fallback: false },
      { flag: FEATURE_FLAGS.snykCodeInlineIgnore, fallback: true },
    ];

    for (const { flag, fallback } of flags) {
      const isEnabled = await this.featureFlagService.fetchFeatureFlag(flag, fallback);
      configuration.setFeatureFlag(flag, isEnabled);
    }
  }
```

Reply:
I tried with allSettle but the code gets a bit more complicated because we need to loop the array of results and assert the `fulfilled` value

Reply:
concurrency should actually be the default here - feel free to push your suggestion :)

Reply:
OMW :)